src/stores/user/index.js

```diff
@@ -9,6 +9,7 @@ export const USER_QUERY = gql`
     currentUser {
       id
       username
+      name
```

Comment:
Can we add comments here, what is the difference between name and username @peratik

Reply:
Added.